docs/docs/references/personas/alia.md

```diff
@@ -0,0 +1,43 @@
+---
+id: alia
+title: Alia
+description: "Data scientist"
+image: '/img/references/alia'
```

Comment:
I was really hoping to get the little persona image on the index cards but this line doesn't actually work... 
<img width="435" alt="image" src="https://github.com/nebari-dev/nebari-docs/assets/15659110/3907bbf1-221b-4201-ad5c-511b938c53a8">

If anyone knows how to customize that little card icon (currently a piece of paper emoji), I'd love to hear it! 

And if we can't find a way to swap it out, then this line should be removed.

Reply:
I haven't figured out anything better. This has been cleaned up as well as it can be. Little paper icon as the default seems to be the only option at this point.